ocs_ci/helpers/helpers.py

```diff
@@ -4439,3 +4441,101 @@ def check_selinux_relabeling(pod_obj):
     key = '"selinuxRelabel": false'
     assert key in output, f"{key} is not present in inspect logs"
     logger.info(f"{key} is present in inspect logs of application pod running node")
+
+
+@retry(
+    (CommandFailed, TimeoutExpired),
+    tries=5,
+    delay=15,
+    backoff=1,
+)
+def download_odf_cli_from_image():
```

Comment:
we should use [registry.redhat.io/odf4/odf-cli-rhel9](http://registry.redhat.io/odf4/odf-cli-rhel9) instead of downloading the binary from the operator pod